### DISCUSSION TITLE
Wrangle Data Sources

### DISCUSSION THREAD

**Discussion by markdav-is** [OpenEugene/fire-refuge#6] [Ideas]
The document we are using now is https://docs.google.com/document/d/e/2PACX-1vSin0jDlJuL55fbj4YvTYIDBAaiihcpcyp60ZKL5KCFNgi8TtQB2OhvqD83jcWVBNRIIiU_mrZl6D1w/pub 

- [ ] COAD data in recovers.org
- [ ] Airtable data 
- [ ] AirBnB integrations

**Comment by markdav-is** [comment]
https://whitebirdclinic.org/holidayfarm

**Comment by thomasfiorelli** [comment]
@nohorse - Should I post the HTML feeds we're currently pulling from for all the real-time public map data (Fire, FEMA shelter locations, etc.)?

**Reply by markdav-is** [reply]
sure!  dump all the link you have, we'll make a plan

**Comment by thomasfiorelli** [comment]
"Mess of Links" - add new sources to this page for review and verification:

https://docs.google.com/document/d/1DxIYD9-JJaV4ju2Cl-6JGVw0hd9PdN35TptGQyOlV6A/edit
